release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1178,6 +1178,17 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-6-known-issues"]
 == Known issues
 
+* Currently, upgrading from {product-title} from 4.5 to {product-version} with Open Virtual Network (OVN) will not work. This will be resolved in a future 4.6.z release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880591[*BZ#1880591*])
+
+* Currently, scaling up {op-system-base-full} worker nodes on an OVN cluster will not work. This will be resolved in a future RHEL 7.8.z and RHEL 7.9.z release.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884323[*BZ#1884323*], link:https://bugzilla.redhat.com/show_bug.cgi?id=1871935[*BZ#1871935*])
```

Comment:
When they're referring to _OVN_ in this context, they mean the OVN-Kubernetes pod network provider. (There are other OVN-based networking stacks, such as the one for OSP.)